app/build.gradle

```diff
@@ -38,4 +38,5 @@ dependencies {
     testImplementation "junit:junit:$rootProject.junitVersion"
     androidTestImplementation "com.android.support.test:runner:$rootProject.testRunnerRulesVersion"
     androidTestImplementation "com.android.support.test.espresso:espresso-core:$rootProject.espressoVersion"
+    implementation files('libs/jsyn-20171016.jar')
```

Comment:
```suggestion
    implementation 'com.github.iamareebjamal:jsyn:16.8.0a'
```

Reply:
@iamareebjamal tried doing that but the build is now failing on my repository. 
![image](https://user-images.githubusercontent.com/22399995/56067769-fc8d2a00-5d99-11e9-8e37-d98cabfee5f9.png)

Reply:
Try 2-3 times

Reply:
Yeah you have to add jotpack as well https://jitpack.io

Reply:
@iamareebjamal the build if failing again(not locally but here)